src/components/ThemeSwitcher.jsx

```diff
@@ -0,0 +1,58 @@
+import { Themes, Label } from './Label';
+import Input from './Input';
+
+const inputProps = [
+  {
+    className: ' checked checked:bg-themeOne-30 checked:hover:bg-themeOne-40',
+    id: 'theme-choice-one',
+    value: 'themeOne',
+    checked: true,
+  },
+  {
+    className: ' checked:bg-themeTwo-30 checked:hover:bg-themeTwo-40',
+    id: 'theme-choice-two',
+    value: 'themeTwo',
+  },
+  {
+    className: ' checked:bg-themeThree-30 checked:hover:bg-themeThree-40',
+    id: 'theme-choice-three',
+    value: 'themeThree',
+  },
+];
+
+const ThemeInputs = () => {
+  return (
+    <>
+      {inputProps.map((props, index) => (
+        <Input key={index} {...props} />
+      ))}
+    </>
+  );
+};
+
+const LabelSelector = () => {
+  return (
+    <>
+      {Themes.map((theme) => (
+        <Label key={theme.id} value={theme.value} htmlFor={theme.id} />
+      ))}
+    </>
+  );
+};
```

Comment:
instead of creating another function that is used only once, please move the code inside the component where its used.